crates/asm-spec/asm.yml

```
@@ -384,6 +405,7 @@ StateRead:
 
             PubVarLen:
               opcode: 0x3C
+              short: PLEN
```

```suggestion
              short: PVLEN
```

just thinking if we have `VAR`, `VLEN`, and `PVAR` maybe this makes more sense?